Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ConnectivityTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.IO;
+using System.Linq;
 using System.Threading;
 using NUnit.Framework;
 using Frends.SFTP.DownloadFiles.Definitions;
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/267e98ab232f82b7a9f5fa1cee119a75ec7687d5/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles.Tests/ConnectivityTests.cs#L43
Please use `Assert.IsTrue` instead of `That`.